docs/fundamentals/aspire-sdk-templates.md

```diff
@@ -109,6 +109,14 @@ dotnet new aspire-starter
 > dotnet new aspire-apphost --framework net8.0
 > ```
 
+You need to trust the ASP.NET Core localhost certificate before running the app. Run the following command:
```

Comment:
Just in case?
```suggestion
You need to trust the ASP.NET Core :::no-loc text="localhost"::: certificate before running the app. Run the following command:
```